### DISCUSSION TITLE
HTTP Code 400 (https://giscus.app/api/discussions)

### DISCUSSION THREAD

**Discussion by yuhangbin** [giscus/giscus#1055] [Q&A] [answered]
curl 'https://giscus.app/api/discussions' \
  -H 'authority: giscus.app' \
  -H 'accept: */*' \
  -H 'accept-language: zh-CN,zh;q=0.9,en;q=0.8,zh-TW;q=0.7' \
  -H 'content-type: application/json' \
  -H 'origin: https://giscus.app' \
  -H 'referer: https://giscus.app/' \
  -H 'sec-ch-ua: "Google Chrome";v="113", "Chromium";v="113", "Not-A.Brand";v="24"' \
  -H 'sec-ch-ua-mobile: ?0' \
  -H 'sec-ch-ua-platform: "macOS"' \
  -H 'sec-fetch-dest: empty' \
  -H 'sec-fetch-mode: cors' \
  -H 'sec-fetch-site: same-origin' \
  -H 'user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36' \
  --data-raw '{"repo":"yuhangbin/yuhangbin.github.io","input":{"repositoryId":"283387947","categoryId":"39529448","title":"posts/why-build-this-site/","body":"# posts/why-build-this-site/\n\n第一次 2017年\n这一次 2023年\n\nhttp://localhost:1313/posts/why-build-this-site/"}}' \
  --compressed

**Comment by laymonage** [comment, marked as answer]
You used the wrong values for repository ID and category ID. Please follow the configuration tool on https://giscus.app to get the values.

**Reply by yuhangbin** [reply]
> You used the wrong values for repository ID and category ID. Please follow the configuration tool on https://giscus.app to get the values.

Sorry, I did find anything tool or explain how to get repository ID and category ID on https://giscus.app.
which repository ID and  category ID is integer value ?

**Reply by yuhangbin** [reply]
I understand now. I need to put my "repository" into the input box, then it will generate the code which contains category ID and repository ID.
